releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,6 +4,9 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v3.0.13 - May, 04 2020
+* Fixed the issue that some reserved symbols misencoded set to environment variables. [#313](https://github.com/Azure/AppConfiguration/issues/313)
```

Comment:
We don't set environment variables. How about this?

Fixed the issue that some symbols, for example, percent sign (%), could not be set to Pipeline Task variables correctly.

Reply:
Yes, should try to avoid `task variable` or `environment variable`.